src/testWithReact.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react';
+
+it('likes react, too!', () => {
+  render(
+    <p>foo bar</p>
+  );
+
+  expect(screen.getByText('foo bar')).toBeInTheDocument();
```

Comment:
The general workflow for react tests is:

1. render the element
2. get the element. preferred way to do this for accessibility is via `screen.getByRole` where we ask it for something like a `heading`, `textbox`, etc (ensures screenreaders/etc controls can find the element, too)
3. expect it to be whatever, in the document, have some content, be enabled/disabled, anything you can imagine